IBAnimatable/Animatable.swift

```diff
@@ -533,12 +525,13 @@ private extension Animatable where Self: UIView {
     })
   }
 
-  func animateOut(x: CGFloat, y: CGFloat, scaleX: CGFloat, scaleY: CGFloat, alpha: CGFloat, completion: AnimatableCompletion? = nil) {
+  // swiftlint:disable function_parameter_count
+  func animateOut(x: CGFloat, y: CGFloat, scaleX: CGFloat, scaleY: CGFloat, alpha: CGFloat, configuration: AnimationConfiguration, completion: AnimatableCompletion? = nil) {
```

Comment:
If `AnimationConfiguration` only be used internal, it is OK to group `x`, `y` etc. inside. Otherwise, we may expose too many details.